selenium_tests/lti_emailer/page_objects/emailer_mainpage_page_object.py

```diff
@@ -55,3 +57,16 @@ def is_loaded(self):
                 'Could not determine if Emailer main page loaded as expected;'
                 'title element was found but did not contain expected text'
             )
+
+    def get_member_link(self):
+        element = self.find_element(*EmailerMainPageLocators.MEMBER_LINK)
+        return element
+
+    def select_member_link(self):
+        """
+        select the course info link element and click it
+        :returns CourseSearchPageObject
+        """
+        self.focus_on_tool_frame()
+        self.find_element(*EmailerMainPageLocators.MEMBER_LINK).click()
+        return CourseEmailerListPage(self._driver)
```

Comment:
This may be just a personal preference thing, but having page objects need to know about other page objects seems like it's going to get messy (it can lead to circular imports, at least).  Why not rely on the test to know that it's transitioned to another page, and to instantiate the new page object there?